src/main/resources/service_config.json

```diff
@@ -298,7 +298,7 @@
     },
     {
       "id": "nfdiv_case_api",
-      "caseTypeId": ["NFD", "NO_FAULT_DIVORCE_BulkAction"],
+      "caseTypeId": ["*"],
```

Comment:
Is there any reason to use '*'? Normally there was a plan to convert the other * ones to explixit caseTypeIds.

Reply:
we're going to have a growing number of case types